docs/api.md

```diff
@@ -1428,7 +1428,7 @@ If `key` is a single character and no modifier keys besides `Shift` are being he
 #### page.route(url, handler)
 - `url` <[string]|[RegExp]|[function]\([string]\):[boolean]> A glob pattern, regex pattern or predicate receiving [URL] to match while routing.
 - `handler` <[function]\([Route], [Request]\)> handler function to route the request.
-- returns: <[Promise]>.
+- returns: <[Promise]<[function]:[Promise]>> Promise which resolves to the function that can be used to remove the route.
```

Comment:
I'm not sure I like returning a function that can remove the route more than explicit `page.removeRoute(url,[handler])`.

Reply:
Agreed, also since there is already a `page.removeListener` for `page.on`, it would make sense to have a `page.removeRoute` also.